tests/library/global-fetch-cookie.spec.ts

```diff
@@ -206,6 +206,28 @@ it('should store cookie from Set-Cookie header even if it contains equal signs',
   });
 });
 
+it('should override cookie from Set-Cookie header even if it expired', async ({ request, server }) => {
+  server.setRoute('/setcookie.html', (req, res) => {
+    res.setHeader('Set-Cookie', [`a=ok`]);
```

Comment:
Please also add a test where the cookie is persistent, i.e. has explicit `Max-Age` or `Expires` attribute in the future.

Reply:
I added a test with expiries in the future. This would also fail before the changes, because `cookieStore` used to keep the old  cookie's expiry.

Reply:
Nice! This does cover the change in the behavior with the expiry time.

What I meant though is to add another test where the original cookie had an explicit expiry time and then new cookie is received with expiry time in the past which should delete the cookie from the storage. Maybe you can just extend the newly added test to cover that?

Reply:
Okay, I extended this test with another cookie that initally has an explicit expiry in the future and later the expiry is set to the past.

Reply:
Thanks!